pycbc/waveform/waveform.py

```diff
@@ -705,6 +716,66 @@ def td_waveform_to_fd_waveform(waveform, out=None, length=None,
     htilde.chirp_length = tChirp
     return htilde
 
+def get_two_pol_waveform_filter(outplus, outcross, template, **kwargs):
+    """Return a frequency domain waveform filter for the specified approximant
+    """
```

Comment:
Please make the doc string more descriptive. Also, stick a note here about the SpinTaylorF2 --> TaylorF2 when the spins are anti-aligned.